### ISSUE TITLE
[Documentation] npm_import and transitive dependencies

### ISSUE THREAD

**Issue by kylepl** [aspect-build/rules_js#1266]
Hi,

I was experimenting with the new ts_proto rules, and it seems I need [npm dependencies](https://github.com/aspect-build/rules_ts/blob/main/docs/proto.md) to get that to work. I don't have any use of npm yet (just starting a Typescript project), so I was trying `npm_import` rather than a lock file, since I'd just have a couple.

But I realized (and correct me if I'm wrong), that it does not give you any of the `npn_import`ed package's dependencies or transitive dependencies - though they can be manually specified right?

If IIUC, I plan to send a PR to just make that clear in the documentation (essentially motivating why it is rarely used).

Thanks!

Kyle

**Comment by alexeagle** [comment]
That is correct `npm_import` does only a single package. PRs to the documentation are most welcome!
